src/client/sandbox/node/element.ts

```diff
@@ -957,7 +957,7 @@ export default class ElementSandbox extends SandboxBase {
         if (!iframe.name)
             return;
 
-        const elementsWithTarget = nativeMethods.querySelectorAll.call(this.document, `*[target="${iframe.name}"]`);
+        const elementsWithTarget = nativeMethods.querySelectorAll.call(this.document, `*[target="${iframe.name.replace(/"/g, '\\"')}"]`);
```

Comment:
For more readability please refactor as follows:
```js
const escapedIframeName  = iframe.name.replace(/"/g, '\\"');
const elementsWithTarget = nativeMethods.querySelectorAll.call(this.document, `*[target="${escapedIframeName}"]`);

```

Reply:
Done